conjur/wrapper/http_wrapper.py

```diff
@@ -63,12 +65,17 @@ def invoke_endpoint(http_verb, endpoint, params, *args, check_errors=True,
                                   auth=auth,
                                   headers=headers)
     except requests.exceptions.SSLError:
-        response = request_method(url, *args,
-                                  params=query,
-                                  verify=ssl_verify,
-                                  auth=auth,
-                                  headers=headers)
-
+        try:
```

Comment:
@mbenita-Cyberark nested try catch bad practice?

Reply:
no problem with doing error handlings inside except statement. infect it is important to make sure except,finally and destructors are non-throws

Reply:
what do you mean?

Reply:
when you have a statement inside "except" clause or "finally" clause you normally don't want it to throw unless you raise explicitly. 
destructors should be non-throw by definition.